core/include/bufr/DataObject.h

```diff
@@ -381,6 +399,23 @@ namespace bufr {
         data_ = data;
       }
 
+      /// \brief Write the data out using a writer.
+      /// \param writer The writer to use.
+      void write(std::shared_ptr<ObjectWriterBase> writer) final
+      {
+        if (auto writerPtr = std::dynamic_pointer_cast<ObjectWriter<T>>(writer))
+        {
+          writerPtr->write(data_);
+        }
+        else
+        {
+          std::ostringstream str;
+          str << "Can't write data of type " << typeid(T).name() << " with writer of type ";
```

Comment:
Somewhat random nitpick, but sometimes apostrophes in log files can create issues with string syntax hilighting. Could you make it (here and other places) "Cannot"?

Reply:
Removed apostrophes in the develop branch. Have you ever run into the syntax highlighting issue yourself?

Reply:
I have, can't recall if it was only in F90 or in general, but I've seen it in vim